rdf/renderers.py

```diff
@@ -22,7 +22,14 @@ class RDFLibRenderer(BaseRenderer):
     """
 
     def render(self, graph, media_type=None, renderer_context=None):
-        return graph.serialize(**self.rdflib_args)
+        view = None
+        if renderer_context is not None and 'view' in renderer_context:
+            view = renderer_context['view']
+        rdflib_args = self.get_rdflib_args(view=view)
+        return graph.serialize(**rdflib_args)
+
+    def get_rdflib_args(self, **kwargs):
+        return self.rdflib_args
```

Comment:
Is this what you were referring to as "clumsy-looking"? In that case I see why you chose to use those words, but I also see why you did it this way. It looks reasonable to me.